### PR TITLE
Split up wheel building for aarch64

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Checklist
- [x] Pull request details were added to CHANGELOG.rst
- [ ] Documentation was updated (if needed)

### CODE REVIEW

**Comment by rhpvorderman** [comment]
@marcelm FYI: I changed the CIBW docker image for linux from manylinux2014 to nothing specified (CIBW default) in this PR. This means the wheels should build on older versions of pip as well from the next release onwards. That should solve a whole category of bugs that was opened on cutadapt/xopen.

**Comment by marcelm** [comment]
Thanks, that is actually something I had wanted to bring up!

**Comment by rhpvorderman** [comment]
I tested it, and it doesn't make any speed difference whether the manylinux1/manylinux2010 wheels are used or the manylinux2014 ones. And since security is not really an issue here (I only use constants and memmove and memcpy from libc) I thought it best to transition.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -200,8 +207,12 @@ jobs:
             {project}/tests/test_compat.py
             {project}/tests/test_igzip_lib.py
       - name: Build sdist
-        if: "runner.os == 'Linux'"
-        run: python setup.py sdist
+        if: "runner.os == 'Linux' and ${{ matrix.build_sdist }}"
+        run: bash -c 'pip install cython && python setup.py sdist'
```
Comment (marcelm) [inline .github/workflows/ci.yml:211]: BTW, the modern way to build an sdist is with [build](https://github.com/pypa/build) with something like "pip install build && python -m build --sdist"

Comment (rhpvorderman) [reply .github/workflows/ci.yml:211]: Thanks for sharing! That is incorporated now!

**Pull request merged**